### PR TITLE
Describe API

### PR DESCRIPTION
**Pull request by agudulin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,23 +14,71 @@ npm install --save dumb-bem
...
 // </div>
 ```
 
+## API
+### `dumbBem(block, extraMakers=[])`
+
+#### Arguments
+
+  - `block` (*String*): name of the base block.
+  - [`extraMakers`] (*Array*): array of functions for adding new class names
```
Comment (robinpokorny) [inline README.md]:
``` diff
- - [`extraMakers`] (*Array*) …
+ - [`extraMakers`] \(*Array*) …
```

File: README.md [hunk 2]
```diff
@@ -14,23 +14,71 @@ npm install --save dumb-bem
...
+  Maker function takes `block` and `props` as arguments and should return anything suitable for [classnames](https://www.npmjs.com/package/classnames) input. E.g. it can be a string, array of string or object.
```
Comment (robinpokorny) [inline README.md]: Link to https://github.com/JedWatson/classnames#usage?

**Comment by robinpokorny** [comment]
👍

**Comment by rafalbromirski** [comment]
👍

**Pull request merged**